src/components/mapPageComponents/mapContainer/ducks/actions.ts

```diff
@@ -0,0 +1,19 @@
+import { genericAsyncActions } from '../../../../utils/asyncRequest';
+import { BlockGeoData, NeighborhoodGeoData } from './types';
+
+export const blockGeoData = genericAsyncActions<BlockGeoData, any>();
+
+export type BlockGeoDataAction =
+  | ReturnType<typeof blockGeoData.loading>
+  | ReturnType<typeof blockGeoData.loaded>
+  | ReturnType<typeof blockGeoData.failed>;
```

Comment:
I think we can combine the `type` to something like `MapActions`. afaik we don't reference the types outside of using them to create the union type C4CAction.